include/sparrow/buffer/u8_buffer.hpp

```diff
@@ -72,17 +78,43 @@ namespace sparrow
         using holder_type = detail::holder<buffer<std::uint8_t>>;
         using buffer_adaptor_type = buffer_adaptor<T, buffer<std::uint8_t>&>;
         using holder_type::extract_storage;
+        using buffer_adaptor_type::operator[];
```

Comment:
This should not be required since the inheritance from `buffer_adaptor` is public. Is that a workaround for a compiler issue?